src/Microsoft.DotNet.Interactive.Formatting/HtmlFormatter{T}.cs

```diff
@@ -246,14 +292,16 @@ static bool BuildTreeView(T source, FormatContext context, MemberAccessor<T>[] m
 
             PocketView view = details[attributes](
                 summary(
-                    span[@class: "dni-code-hint"](code)),
+                    span[@class: "dni-code-hint"](summaryContent)),
                 div(
                     Html.Table(
                         headers: null,
-                        rows: memberAccessors.Select(
+                        rows: rows.Select(
                             a => (IHtmlContent)
```

Comment:
minor: Consider renaming `a` to `row`

Reply:
Never mind - looks like we also have other instances of `a` and `c` above and after staring at the code a bit longer it is clearer that they stand for `accessor` and `context` respectively :)